conf/modules/mobsterh.config

```diff
@@ -0,0 +1,30 @@
+process {
+    withName: 'MOBSTERh' {
```

Comment:
Why does this have a lowercase h? We generally use all upper case for module names.